test/python/sdpa/frost/test_sdpa_fwd_mxfp8_sm100.py

```
@@ -349,3 +368,265 @@ def _sf(dims):
+@pytest.mark.L0
+@pytest.mark.parametrize("in_key", _INS)
+@pytest.mark.parametrize("causal", [False, True])
+@torch_fork_set_rng(seed=0)
+def test_mxfp8_dense_padding(in_key, causal):
+    """Dense KV padding mask (frontend extension on sdpa_mxfp8): batch 0 uses all
+    256 KV cols, batch 1 only 192 (a partial KV tile). Padded columns must not
+    leak into the softmax or the in-kernel Amax_O. Stats stay off (padded+stats
+    needs the per-batch LSE trim, which this cell does not declare)."""
+    scale = 1.0 / math.sqrt(128)
+    sk = dict(use_causal_mask=True) if causal else {}
+    O, O_ref, amax = _run(2, 8, 8, 256, in_key, torch.float16, scale=scale, sdpa_kwargs=sk, seq_lens_kv=[256, 192], stats=False)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)